bimcollab_zoom/com.github.wycomco.bimcollab_zoom.munki.recipe

```diff
@@ -8,37 +8,42 @@
 	<string>com.github.wycomco.bimcollab_zoom.munki</string>
 	<key>Input</key>
 	<dict>
+		<key>APPLICATIONS_PATH</key>
+		<string>/Applications</string>
 		<key>MUNKI_REPO_SUBDIR</key>
 		<string>apps/%NAME%</string>
```

Comment:
I think it it more common to put simple apps directly into the apps directory, especially for apps which do not get updates that frequently.